packages/react-dom/src/server/ReactDOMFizzServerNode.js

```diff
@@ -189,8 +190,15 @@ function renderIntoContainerAsPipeableStream(
   };
 }
 
+let renderIntoContainerExport:
+  | void
+  | typeof renderIntoContainerAsPipeableStream;
+if (enableFizzIntoContainer) {
+  renderIntoContainerExport = renderIntoContainerAsPipeableStream;
```

Comment:
It would be more idiomatic to create a `ReactDOMLegacyServerNode.stable.js` file that re-exports an inner one but excludes this export name.

Otherwise this bundle will still contain an exported name whose value is `undefined`.

This gets more important if/when we adopt ESM since a missing name is an early error.